plugins/faustwp/includes/rest/callbacks.php

```diff
@@ -76,6 +76,17 @@ function register_rest_routes() {
 			'permission_callback' => __NAMESPACE__ . '\\rest_authorize_permission_callback',
 		)
 	);
+
+	// Deprecated. Faust.js packages now use faustwp/v1/authorize.
```

Comment:
We should use the PHPDoc `@deprecated` tag here as well so we can easily find all deprecations.

Reply:
I didn't use the tag initially since PHPDoc doesn't support comments and tags inside of functions, but there's nothing stopping us from doing it anyway. It looks nicer and is more consistent. Addressed in 510034c